slo/rules.go

```diff
@@ -26,6 +26,10 @@ type MultiBurnRateAlert struct {
 }
 
 func (o Objective) Alerts() ([]MultiBurnRateAlert, error) {
+	if o.Alerting.Disabled {
+		return nil, nil
+	}
+	
```

Comment:
Not sure if I missed something, but while taking a look at this I noticed that this function is not called at all in the whole codebase, except for the test [TestObjective_Alerts](https://github.com/pyrra-dev/pyrra/blob/62480166e2cec1b0d2e985f9a27c6c1f8fc4f9e9/slo/rules_test.go#L1217).

PrometheusRules CRs were only generated without the alerting rules when I made changes to the func  `Burnrates()` , so I'm not sure why the function `Alerts()` exists at the moment 🤔